threeplay.go

```diff
@@ -36,6 +38,18 @@ type File struct {
 	ErrorDescription     string `json:"error_description"`
 }
 
+type FilesPage struct {
```

Comment:
We usually split and then group the structs by "topics" or something like that. We did something similar to that on encoding-wrapper packages: https://github.com/NYTimes/encoding-wrapper/tree/master/encodingcom.

In this case, you could move file related structs (FilesPage and File) and operations to a "file.go" file.